tutorials/applications/recipes/loading_text_files.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>So there are two interfaces for Fugue with the <code>transform()</code> and <code>FugueWorkflow()</code> . Of course if using the <code>transform()</code>, you don't need to do this because you can just load in the file and pass it. So this is right tin that it only makes sense for <code>FugueWorkflow()</code>.</p><p> </p><p>For <code>FugueWorkflow()</code>, can we use the context manager syntax <code>with ..</code> ? That is the syntax we are pushing for now, but I'm curious to hear if you prefer this style.         </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Cool.  I like <code>FugueWorkflow</code> &amp; haven't used <code>transform</code> as I typically want <code>Fugue</code> for multiple tasks.  Do you want me to mention <code>transform</code> here via example?    </p>

Reply:
<p>No need to mention transform since the user brings in their own loader in that case so they don't run into this problem. </p><p> </p><p>In the next Fugue version, we'll have the following syntax:</p><p> </p><pre class="ql-syntax" spellcheck="false">with FugueWorkflow() as dag:
    df = dag.load(file_path)
    df = df.take(5)
    df.save(out_path)

# Separation of build and run like Prefect
dag.run(SparkExecutionEngine)
dag.run("spark")    # alternative specification
</pre>

Reply:
<p>Just add it in however you want and I can take there of making the edits once this new changed in released though.</p>

Reply:
Gotcha, I do like the second syntax, though I'm not sure if I have a strong preference one way or the other.  I'm happy to help update on the next `fugue` version, just linkedin message or slack me